index.html

```diff
@@ -738,6 +759,136 @@ <h2 class="feedback__title title">Great Feedback From Candidates</h2>
                 </div>
               </div>
             </div>
+
+            <!-- Feedback card -->
+            <div class="feedback-wrapper__card feedback-card">
+              <div class="feedback-card__title">
+                <img
+                  class="feedback-card__quotes"
+                  src="./src/icons/double-quotes.svg"
+                  alt="double quotes"
+                />
+
+                <img
+                  class="feedback-card__quotes-active"
+                  src="./src/icons/double-quotes-active.svg"
+                  alt="double quotes"
+                />
+
+                <p class="feedback-card__paragraph">
+                  Very decent services & care about my health. Really happy with
+                  his advice. Lorem Ipsum is simply dummy text of the printing
+                  and typesetting industry Very decent services & care about my
+                  health. Really happy with his advice.
+                </p>
+              </div>
+
+              <div class="feedback-card__user-preview user-preview">
```

Comment:
```suggestion
              <div class="feedback-card__user-preview">
```
As I see in your code, the `user-preview` class doesn't exist.

Reply:
yes, thanks you, deleted useless class